api/v1/endpoints.js

```diff
@@ -2,7 +2,7 @@ var requests = require('./requests');
 
 var endpoints = {};
 
-endpoints['/v1/user/attendee'] = {
+endpoints['/v1/user/'] = {
```

Comment:
It seems like the `/v1/user/` endpoint isn't updated on the wiki, can you update it?